etc/sync-function-validation-module.js

```diff
@@ -41,44 +41,23 @@ function() {
   function validateDoc(doc, oldDoc, docDefinition, docType) {
     var validationErrors = [ ];
 
-    validateImmutableDoc(doc, oldDoc, docDefinition, validationErrors);
```

Comment:
This file was refactored to make it easier to share state (e.g. `validationErrors`, `itemStack`, `attachmentReferenceValidators`) between validation functions. I highly recommend viewing this file's changes with `?w=1` to make it more manageable.